packages/bpk-component-calendar/src/BpkCalendarDate.js

```diff
@@ -155,6 +155,8 @@ BpkCalendarDate.propTypes = {
   preventKeyboardFocus: PropTypes.bool,
 };
 
+BpkCalendarDate.propTypes = propTypes;
```

Comment:
Out of interest, whats the advantage of having a separate `propTypes` object?  Why not follow the same pattern for defaultProps or vice versa?

Reply:
Since this component uses all the same props as `BpkCalendarDate` I was advised to export those to avoid duplication as much as possible

Reply:
@emmajam There are babel extensions that strip out `propTypes` and if you try to do `...Component.propTypes` that can cause issues with that. Having a dedicated object helps we've found.

Reply:
Great question, see this rule https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/forbid-foreign-prop-types.md 👍

Reply:
Thanks guys ❤️  sorry for delayed response!